cliquet/tests/support.py

```diff
@@ -87,12 +87,9 @@ class BaseWebTest(FakeAuthentMixin):
 
     api_prefix = "v0"
 
-    def get_test_app(self):
```

Comment:
Before #193 it was not used and was executed by nose twice per test suite (because contains "test" in name)

I will restore it and prefix it with ``_``.